src/reason-parser/reason_parser.mly

```diff
@@ -1047,6 +1047,9 @@ let record_pat_spread_msg =
 Explanation: you can't collect a subset of a record's field into its own record, since a record needs an explicit declaration and that subset wouldn't have one.
 Solution: you need to pull out each field you want explicitly."
 
+let lowercase_module_msg s =
+  Printf.sprintf "Module names must be uppercase identifiers. `%s` is not an uppercase identifier." s
```

Comment:
Do we have `uppercase identifier` defined somewhere?  It may be helpful to be explicit here - something along the lines of ``Module names must start with an uppercase letter.  `%s` does not start with an uppercase letter.``

Reply:
@hcarty that sounds good to me. I'm expecting @chenglou to bikeshed the error message anyway!